test/util/create-files

```diff
@@ -68,6 +68,6 @@ show_summary()
 # main
 #
 
-get_args $*
+get_args "$@"
```

Comment:
Why prefer `$@` over `$*`?

Reply:
either `$@` or `$*` will work in this case but it should be doublequoted because the unquoted version has undefined behavior